SystemDocumentation/SystemDocs.txt

```diff
@@ -0,0 +1,206 @@
+This is the system documentation for our POS System. It will cover all the files in our system and go one by one and describing what they do.
+
+In the root directory you will find these files that contribute to our project.
+
+Database.py -
+Overview: This is the Database class that we use throughout the project that connects our code to the database, we are using heroku's database tools
+to connect this class to the DB
+
+Use Cases:
+We have our Get queries. These queries get information from the database, and don't write any information from the database. They are
+named exactly what they are supposed to do.
+
+We have our User queries, these queries interact with user data and can write and read information from the database.
+
+We have our Insert queries, these queries insert information into the database, it will write transaction, and put new information into
+the database.
+
+We have our Update queries, these queries don't insert any new rows into the database, but they will update existing rows in the
+database.
+
+DatabaseUrlGenerator.py -
+Overview: This file will get the database url for local development and for production development so we don't have that in our codebase
+
+Use Cases:
+getLines(): This will get database info from a text file on production.
+
+getDatabaseUrl(): This will grab the database url from the text file and actually use it in the DbConnection.py file
```

Comment:
wording is a bit strange here. As noted above, the env var gets used on prod and the file gets used locally